lua/entities/emplacements_turret_base/init.lua

```diff
@@ -120,8 +120,19 @@ function ENT:GetDesiredShootPos()
     return shootTrace.HitPos
 end
 
+function ENT:ApplyRecoil( randomMul, recoilMul, finalMul )
+    if not self:IsValid() then return end
+    
+    local randomComponent = VectorRand( -1, 1 ) * randomMul
+    local recoilComponent = self:GetRight() * recoilMul 
+    local finalForce      = ( randomComponent + recoilComponent ) * finalMul
+    
+    self:GetPhysicsObject():ApplyForceCenter( finalForce )
+end
+
 function ENT:PhysicsSimulate( phys, deltatime )
     phys:Wake()
+    if not IsValid( self.turretBase ) then return end
```

Comment:
Why was this line added?

Reply:
kept getting an error, related to the code below this running w/o a turretBase